packages/rlc-common/src/buildResponseTypes.ts

```diff
@@ -86,7 +85,11 @@ export function buildResponseTypes(model: RLCModel) {
     responsesFile.addImportDeclarations([
       {
         namedImports: ["RawHttpHeaders"],
-        moduleSpecifier: "@azure/core-rest-pipeline"
+        moduleSpecifier:
```

Comment:
I saw similar code repeating multiple time, can we refactor this part?

Reply:
Let me build a helper for this.

Reply:
updated